src/Label.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import cx from 'classnames';
+
+/**
+ * @module Label
+ */
+class Label extends React.Component {
+	render() {
+		const {
+			inputId,
+			text,
+			isRequired,
+			children,
+			className,
+			...other
+		} = this.props;
+
+		const classNames = cx(
+			{ required : isRequired },
+			className
+		);
+
+		return (
+			<label className={classNames}
+				htmlFor={inputId}
+				{...other}>
+				{text}
+				{children}
+			</label>
+		);
+	}
+}
+
+Label.propTypes = {
+};
+
+export default Label;
```

Comment:
it looks like `<Label>` doesn't do much more than a regular `<label>` - it just changes the name `htmlFor` attribute to `inputId` and adds a switch for the `.required` classname.

IMO, the attribute/prop renaming obscures the implementation more than necessary, and at that point, writing a component for the sake of a className on an element is probably overkill